.github/workflows/main.yaml

```diff
@@ -25,6 +25,7 @@ env:
   # /make/builder.Dockerfile
   # /taprpc/Dockerfile
   # /tools/Dockerfile
+  # /.github/workflows/*.yml
```

Comment:
This line should be added to all the other spots where we specify a go version for a Dockerfile (the other files mentioned in the comment itself).

Reply:
I've added linters to check the Go version on our dockerfile files and YAML files. I've removed the consistency check comments.